packages/ui-components/src/components/svg-icon/readme.md

```diff
@@ -100,22 +101,6 @@ export const SvgIconExample: React.FC = () => (
 | `"icon"` | The icon container. |
 
 
-## CSS Custom Properties
```

Comment:
why is this removed?

Reply:
Ups, don't know, I didn't notice, probably something went wrong when generating the readme.

I will reset this on the upcoming commit.